adapters/criteo/criteo_test.go

```diff
@@ -2,13 +2,14 @@ package criteo
 
 import (
 	"fmt"
+	"testing"
```

Comment:
Noting +2, -1 changed lines here that are unrelated to the version package update.